src/main/scala/collatz/Collatz.scala

```diff
@@ -1 +1,13 @@
+def collatz(n: Int): Int = 
+    if n % 2 == 1 then 3*n+1
+        else n/2
 
+
```

Comment:
Interesting implementation however, it is much simpler to use a recursion technique like in the sample solution